candidate/views_admin.py

```diff
@@ -2000,61 +2004,61 @@ def candidate_edit_view(request, candidate_id=0, candidate_we_vote_id=""):
             'ballotpedia_candidate_id':         ballotpedia_candidate_id,
             'ballotpedia_candidate_id_dict':              
             {
-                'label': 'Candidate Name (from Ballotpedia)',
-                'id': 'ballotpedia_candidate_id_id',
-                'name': 'ballotpedia_candidate_id',
-                'value': ballotpedia_candidate_id if ballotpedia_candidate_id else candidate_on_stage.ballotpedia_candidate_id
-			},
```

Comment:
FYI these are inline with open brackets, just doesn't look like it on the diff for some reason